modules/waypoints_dict_to_list.py

```diff
@@ -0,0 +1,27 @@
+"""
+Module to convert waypoints dict to waypoints list, and generate kml file.
+"""
+
+def waypoints_dict_to_list(waypoint_name_to_coordinates: "dict[str, tuple[float, float]]") -> "tuple[bool, list[tuple[float,float]]]":
```

Comment:
Formatting: Put return type on new line.

Reply:
Sorry could you explain this one a little more?

Reply:
This line is too long, so it needs to be split into 2 lines. The style guide has an example: https://uwarg-docs.atlassian.net/wiki/spaces/CV/pages/2253226033/Python+Style+Guide